components/resources/ResourceDisplay.js

```diff
@@ -5,11 +5,15 @@ import Rating from "@mui/material/Rating";
 import CardActions from "@mui/material/CardActions";
 import CardContent from "@mui/material/CardContent";
 import Typography from "@mui/material/Typography";
+import PreviousPage from "../common/PreviousPage";
 
 function ResourceDisplay({ resource }) {
+  const goBackPageText = "Back to Learning Unit";
```

Comment:
Same here

Reply:
Done!
Also, I changed the native button with the component `Button` from Material UI.